### ISSUE TITLE
v1.3.1 causes "Uncaught SyntaxError: Unexpected token :" error

### ISSUE THREAD

**Issue by jeanduplessis** [ef4/fast-sourcemap-concat#53]
I'm using `ember-auto-import` which depends on `broccoli-concat` which depends on this package.

Recently did a reinstall of my node_modules packages causing the latest versions to be pulled in.

Since then I get the following exception:
```
Uncaught SyntaxError: Unexpected token :
    at Object../node_modules/showdown/dist/showdown.js (app.js:417)
    at __webpack_require__ (app.js:80)
    at Module.eval [as callback] (app.js:21)
    at Module.exports (loader.js:106)
    at Module._reify (loader.js:143)
    at Module.reify (loader.js:130)
    at Module.exports (loader.js:104)
    at Module._reify (loader.js:143)
    at Module.reify (loader.js:130)
    at Module.exports (loader.js:104)
```

Was able to work around this issue by locking this package to 1.3.0 with the following in my package.json.
```
"resolutions": {
    "fast-sourcemap-concat": "1.3.0"
}
```

**Comment by ef4** [comment]
I think this is a bug in source-map-url 0.4.0, introduced [here](https://github.com/lydell/source-map-url/commit/0ec3e326fd128154e6ad0610d82abdc284c40b05). It now matches too broadly and results in invalid Javascript.

**Comment by ef4** [comment]
Fixed by #54
